Detectors/TRD/calibration/workflow/trd-dcs-sim-workflow.cxx

```diff
@@ -15,38 +15,36 @@
 o2::framework::WorkflowSpec defineDataProcessing(o2::framework::ConfigContext const& configcontext)
 {
   std::vector<o2::dcs::test::HintType> dphints;
-  // for TRD
-  // for test, we use less DPs than official ones
+  // for TRD, official list
+
+  // gas parameters
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gasCO2", 0, 50.}); // adding a data point of type double with the name trd_gasCO2 which takes values between 0 and 50
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gasH2O", 0, 500.});
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gasO2", 0, 100.});
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gaschromatographCO2", 0, 100.});
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gaschromatographN2", 0, 100.});
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gaschromatographXe", 0, 100.});
-  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_hvAnodeImon[00..02]", 0, 50.});
-  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_hvAnodeUmon[00..02]", 0, 50.});
-  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_hvDriftImon[00..02]", 0, 50.});
-  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_hvDriftUmon[00..02]", 0, 50.});
-  // dphints.emplace_back(o2::dcs::test::DataPointHint<int32_t>{"trd_fedChamberStatus", 0, 255});
-  dphints.emplace_back(o2::dcs::test::DataPointHint<int32_t>{"trd_runNo", 0, 255});
-  dphints.emplace_back(o2::dcs::test::DataPointHint<int32_t>{"trd_runType", 0, 255});
-  // for TRD, official list, not yet complete
-  /*
-  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gasCO2", 0, 50.});
-  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gasH2O", 0, 50.});
-  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gasO2", 0, 50.});
-  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gaschromatographCO2", 0, 50.});
-  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gaschromatographN2", 0, 50.});
-  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gaschromatographXe", 0, 50.});
+
+  // HV parameters
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_hvAnodeImon[00..539]", 0, 50.});
```

Comment:
```suggestion
  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_hvAnodeImon[00..539]", 0, 20.e-9});
```

Reply:
Ok, if we have the prefix units, it's probably ok. Only if we want to distinguish what's where, but the key test will anyways be with real data.